resources/marketplace/extension-page.md

```diff
@@ -0,0 +1,166 @@
+1. Expand the AWS Region that contains the resource that you want to manage. (To hide or show AWS Region information, choose **View, Command Palette** on the menu bar, and then choose **AWS: Hide region from the Explorer** or **AWS: Show region in the Explorer**.)
+1. Expand the AWS service that contains the resource that you want to manage. (Currently, the Toolkit doesn't support all available AWS services and resource types.)
+1. If applicable, expand the parent resource that contains the child resource that you want to manage.
+1. Right-click the resource to manage, and then choose one of the available actions in the context menu. (Currently, the Toolkit doesn't support all available resource actions.)
+
+For more information, see the following in the _AWS Toolkit for Visual Studio Code User Guide_:
+
+* [Setting Up Your AWS Credentials](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/setup-credentials.html)
+* [Changing AWS Regions](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/setup-region.html)
+* [Exploring AWS Explorer](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/aws-explorer.html)
+
+[Return to Top](#top)
+
+### <a id="developing-serverless-applications"></a>Developing Serverless Applications
+
+You can use the AWS Toolkit for Visual Studio Code to create, run, debug, and deploy serverless applications.
+
+* [Create a Serverless Application](#create-serverless-application)
+* [Run or Debug a Serverless Application](#run-debug-serverless-application)
+* [Deploy a Serverless Application](#deploy-serverless-application)
+
+[Return to Top](#top)
+
+#### <a id="create-serverless-application"></a>Create a Serverless Application
+
+1. On the menu bar, choose **View, Command Palette**.
+1. Begin typing: `AWS: Create new SAM Application`. When you see the **AWS: Create new SAM Application** command, choose it.
+1. Follow the on-screen instructions to finish creating the SAM application.
+
+The Toolkit produces the application code and files, adds them to the location you specified, and opens the program file in the editor.
+
+For more information, see [Creating a Serverless Application](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/create-sam.html) in the _AWS Toolkit for Visual Studio Code User Guide_.
+
+[Return to Top](#top)
+
+#### <a id="run-debug-serverless-application"></a>Run or Debug a Serverless Application
+
+In the **AWS: Explorer** Side Bar, after you choose the **AWS** icon in the Activity Bar, CodeLenses display within open serverless application code files above functions that use AWS Lambda function handler syntax. (A _handler_ is a function that Lambda calls to start execution of a Lambda function.) These CodeLenses enable you to run or debug the corresponding serverless application locally. CodeLens actions here include:
```

Comment:
`In the **AWS: Explorer** Side Bar, after you choose the **AWS** icon in the Activity Bar` 
->
'Clicking on the **AWS** icon in the Activity Bar activates the Toolkit. Once the toolkit is activated, CodeLenses display...`

Reply:
Removed "In the AWS: Explorer Side Bar..."